src/components/GoogleMapContainer.ts

```diff
@@ -179,7 +179,7 @@ class GoogleMapContainer extends Component<GoogleMapContainerProps, { alertMessa
     private fetchLocationsByMicroflow(microflow: string, contextObject?: mendix.lib.MxObject) {
         if (microflow) {
             window.mx.ui.action(microflow, {
-                callback: (mxObjects: mendix.lib.MxObject[]) => this.setLocationsFromMxObjects(mxObjects),
+                callback: mxObjects => this.setLocationsFromMxObjects(mxObjects as mendix.lib.MxObject[]),
```

Comment:
I think this `callback: (mxObjects: mendix.lib.MxObject[]) => this.setLocationsFromMxObjects(mxObjects)`  is preferable

Reply:
Yes preferable, trough it was have an error. does this shows in you code too?